src/core/dfn.js

```diff
@@ -32,3 +46,27 @@ export function run(conf) {
       }, $dfn);
   });
 }
+
+function autoPluralizeDfns() {
+  const links = [...document.querySelectorAll("a")];
+  const dfns = [...document.querySelectorAll("dfn")];
+
+  return function shouldPluralize(dfn) {
+    const plural = pluralize(norm(dfn.textContent));
+    if (
+      // has overridden to not do pluralization
+      dfn.hasAttribute("data-lt-noPlural") ||
+      // no <a> references the plural term
+      !links.find(({ textContent }) => norm(textContent) === plural) ||
```

Comment:
So, I think we want to reverse the logic here, because all "or"s needs to all be evaluated... what we want is "and" ... "and"... `return true`, so we do less work. Make sense?